### PR TITLE
Fixing #1723

### PR DESCRIPTION
**Pull request by sithhell**
This PR ticks of the remaining points of #1723.

### CODE REVIEW

File: docs/manual/build_system/prerequisites.qbk [hunk 1]
```diff
@@ -73,6 +73,11 @@ favorite compiler with __hpx__ visit __stellar_hpx_buildbot__.
       for compiling __hpx__ if you use gcc V4.6.x. Please use either
       an earlier or a later version of Boost with this compiler.]
 
+[important When compiling with the Intel Compiler on Linux systems, we only support
+      C++ Standard Libraries provided by gcc 4.6 and upwards. If the 'g++' in your
+      path is something below that requirement, please add this to your CMAKE_CXX_FLAGS
+      with the `-gxx-name=/path/to/g++` flag.]
+
```
Comment (hkaiser) [inline docs/manual/build_system/prerequisites.qbk]: What do you mean by 'add 'this' ? The correct version of g++? the library?

Comment (sithhell) [inline docs/manual/build_system/prerequisites.qbk]: On 10/21/2015 01:20 PM, Hartmut Kaiser wrote:
> In docs/manual/build_system/prerequisites.qbk
> <https://github.com/STEllAR-GROUP/hpx/pull/1808#discussion_r42610928>:
> 
>> @@ -73,6 +73,11 @@ favorite compiler with __hpx__ visit __stellar_hpx_buildbot__.
>>        for compiling __hpx__ if you use gcc V4.6.x. Please use either
>>        an earlier or a later version of Boost with this compiler.]
>>  
>> +[important When compiling with the Intel Compiler on Linux systems, we only support
>> +      C++ Standard Libraries provided by gcc 4.6 and upwards. If the 'g++' in your
>> +      path is something below that requirement, please add this to your CMAKE_CXX_FLAGS
>> +      with the `-gxx-name=/path/to/g++` flag.]
>> +
> 
> What do you mean by 'add 'this' ? The correct version of g++? the library?

'this' was meant to refer to the CMAKE_CXX_FLAGS, do you have a better
formulation for this?

> 
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/STEllAR-GROUP/hpx/pull/1808/files#r42610928>.
> 


-- 
Thomas Heller
Friedrich-Alexander-Universität Erlangen-Nürnberg
Department Informatik - Lehrstuhl Rechnerarchitektur
Martensstr. 3
91058 Erlangen
Tel.: 09131/85-27018
Fax:  09131/85-27912
Email: thomas.heller@cs.fau.de

Comment (hkaiser) [inline docs/manual/build_system/prerequisites.qbk]: Ok, then the question is more that you should clarify what `-gxx-name=/path/to/g++`  should point to? I assume a g++ of version >= 4.6. That's not really clear, at least it was not to me.

Comment (gentryx) [inline docs/manual/build_system/prerequisites.qbk]: How about this?

      Important: when compiling with the Intel Compiler on Linux systems, we only support
      C++ Standard Libraries provided by gcc 4.6 and upwards. If the 'g++' in your
      path is older than 4.6, please specify the path of a newer g++ by setting
      CMAKE_CXX_FLAGS='-gxx-name=/path/to/g++' via cmake.

Comment (hkaiser) [inline docs/manual/build_system/prerequisites.qbk]: LGTM.

**Pull request merged**